stage1/tasks/hangman/README.md

```diff
@@ -0,0 +1,106 @@
+- Initially, body in the index.html file _must be empty_, all necessary elements are generated using JS.
+- At the beginning of the game, the question-answer pair must be _chosen randomly_.
+- The application can be done _in English or in Russian_.
+
+## Repository requirements
+
+- The task must be done in private school's repository.
+- The code must be committed to a separate branch named as the task.
+- The `main` (or old style master) branch must be empty (contain only files like README.md or .gitignore).
+- Commit messages must follow the [guideline](https://docs.rs.school/#/git-convention)
+- Once the work is finished, create a pull request from a **development** branch to **main** (or master).
+- The pull request name must contain **the task name**.
+- The pull request description must be provided according to the [template](https://docs.rs.school/#/pull-request-review-process?id=Требования-к-pull-request-pr).
+
+## Technical requirements
+
+- The application is displayed correctly and functions properly in the latest version of Google Chrome.
+- It is **not allowed** to use:
+  - CSS frameworks (e.g., `Bootstrap`),
+  - JS frameworks (e.g., `Angular`, `React`, `Vue`, etc.),
+  - Outdated libraries (e.g. `JQuery`, etc.),
+  - `TypeScript`.
+- You can use CSS preprocessors (`SASS`, `SCSS`) and `normalize.css`.
+- Using JS frameworks (e.g., Angular, React, Vue, etc.) is not allowed.
+
+## CrossCheck Criteria (150 points)
+
+_It is recommended to print the secret word in the browser's console to facilitate the cross-check process._
+
+- [ ] Responsive/adaptive UI from 1440px to 360px viewport: `+10`
+- [ ] The generation of DOM elements is implemented. `body` in the `index.html` is empty (can contain only `script` tag): `+20`
+- [ ] The game starts with the correct default view (empty gallows, underscores for secret word, etc.) and a random question: `+5`
+- [ ] The user can play the game by using the virtual keyboard: `+20`
+- [ ] The user can play the game by using the physical keyboard: `+20`
+- [ ] When the letter is correct, it appears instead of the corresponding blank line. If the letter repeats in the word, all corresponding underscores must be replaced by it: `+15`
+- [ ] When the letter is incorrect:
+  - [ ] the incorrect guesses counter is updated: `+5`
+  - [ ] a body part is added to the gallows: `+10`
+- [ ] The clicked/pressed letter is disabled: `+5`
+- [ ] The body parts appear on the gallows in the logical order (head, body, left arm, right arm, left leg, right leg): `+5`
+- [ ] When the user runs out of 6 attempts or wins the game, the modal window appears: `+10`
+- [ ] The modal window includes the message about the game's outcome (winning or losing), the secret word and the 'play again' button: `+10`
+- [ ] When the user clicks the 'play again' button, the game starts over by showing a new question and resetting the gallows, the incorrect guesses counter and the underscores for the secret word: `+15`
+
```

Comment:
I didn't see any information (sorry if I missed) about how many words should be in vocabuary bank? For example, is it OK if student implement only one word and it'll be the key word for all games? Or, for example, student has to add 10 words and it'll appear randomly..?

Reply:
@helenakrasnova I haven't set the minimum amount of question-word pairs, but there is a requirement about the randomly-chosen question at the beginning of the game.

I can add a must-have that that the secret word should be always different (thanks for the tip).

Regarding the min number, I think it's more a suggestion to use at least 10 question-answer pairs, because I'm not sure how the students can check this amount.

Reply:
done